TestingApp/ViewController/ViewController.swift

```diff
@@ -155,6 +155,7 @@ class ViewController: UIViewController {
 
 extension ViewController {
     func presentSettings() {
+        Glia.sharedInstance.minimize()
```

Comment:
What is the purpose of this here?

Reply:
Thank you, good catch. It's needed to minimize engagement view before handling `glia://widgets/settings` deep link.
I moved to more suitable place, SettingsDeeplingHandler. 95891e12de8d30c53a07767fbcca81b62bf4e64d